src/main/java/seedu/contax/storage/JsonSerializableAddressBook.java

```diff
@@ -49,21 +49,32 @@ public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
      *
      * @throws IllegalValueException if there were any data constraints violated.
      */
-    public AddressBook toModelType() throws IllegalValueException {
+    public AddressBook toModelType() {
         AddressBook addressBook = new AddressBook();
 
         for (JsonAdaptedTag jsonAdaptedTag: tags) {
-            Tag tag = jsonAdaptedTag.toModelType();
-            if (addressBook.hasTag(tag)) {
-                throw new IllegalValueException(MESSAGE_DUPLICATE_TAG);
+            Tag tag;
+            try {
+                tag = jsonAdaptedTag.toModelType();
+                if (addressBook.hasTag(tag)) {
+                    throw new IllegalValueException(MESSAGE_DUPLICATE_TAG);
+                }
+            } catch (IllegalValueException e) {
+                continue;
```

Comment:
Can you add a `logger.fine()` call that says a tag was skipped.

Reply:
thanks for the suggestion, added in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/256#:~:text=Add%20logging%20of%20skipped%20entries